elementary/monitor/data_monitoring.py

```diff
@@ -130,6 +130,7 @@ def generate_report(self, tracking: AnonymousTracking, days_back: Optional[int]
         bool, str]:
         now_utc = get_now_utc_str()
         html_path = self._get_report_file_path(now_utc, file_path)
+        os.makedirs(os.path.dirname(html_path), exist_ok=True)
```

Comment:
What do you think about using `makedirs` in the config object? this way if we add more files to the target dir we don't need to remember calling this code in every flow

Reply:
Good point, done